### DISCUSSION TITLE
Add -- Option: wine-wayland && Option: disable runtime-updates

### DISCUSSION THREAD

**Discussion by slynobody** [lutris/lutris#3666] [Ideas]
Hi,

a request for 2 new options:

1) integrate an option to disable the runtime-updates
each time the client starts a series of tries to update runtimes  are performed. you can stop this by pressing escape (i stumbled upon this, there is no text about it). 
if you have no keyboard around (f.e. lutris runnig through kodi etc.) and/or are offline the time until the client starts as well as each game chosen started (until the update-procedure accepts a failure or has reached a time-limit, i did not take a look)  is a *serious* time-waste. you minimally *should* integrate an option to disable the update-checks if you force your users to tolerate this checks *each single time* the client starts or a game is chosen. 

2) integrate wine-wayland
name says it all, seems to be one way to the long-term-future of wine. should be integrated.

Thank you.

**Comment by slynobody** [comment]
anything new on this? a feature to turn off icon-fetching at startup  (as well as theming like in the latest heroic-version) would be great too.

**Comment by strycore** [comment]
It's been possible to skip the runtime / media update for a while. Run Lutris with LUTRIS_SKIP_INIT=1. 
This is already the case when creating Lutris shortcuts.

I have no idea what you mean by Wine-wayland integration. Installed Wine versions should be detected. Just use Wine-Wayland if that's your jam, I don't get what Lutris needs to do.

**Comment by strycore** [comment]
>  you can stop this by pressing escape 

This shouldn't be possible. The close button was removed for a reason.
